app/src/main/java/co/icanteach/projectx/data/feed/MoviesLocalDataSource.kt

```diff
@@ -0,0 +1,13 @@
+package co.icanteach.projectx.data.feed
+
+import co.icanteach.projectx.data.local.dao.PopularTVShowDao
+import co.icanteach.projectx.data.local.entity.PopularTVShowItemEntity
+import javax.inject.Inject
+
+class MoviesLocalDataSource @Inject constructor(private val popularTVShowDao: PopularTVShowDao) {
+
+    fun fetchMovies() = popularTVShowDao.getPopularTVShowsEntity()
```

Comment:
as I want to provide a naming standart on the codebase, it is better to use fetch prefix for remote datasource methods and get for local datasource methods